relayer/processor/path_processor_internal.go

```diff
@@ -558,36 +559,41 @@ func (pp *PathProcessor) processLatestMessages(ctx context.Context, messageLifec
 	var eg errgroup.Group
 	eg.Go(func() error {
 		if err := pp.assembleAndSendMessages(ctx, pp.pathEnd2, pp.pathEnd1, pathEnd1Messages); err != nil {
-			pp.log.Error("Error sending messages",
-				zap.String("src_chain_id", pp.pathEnd1.info.ChainID),
-				zap.String("dst_chain_id", pp.pathEnd2.info.ChainID),
-				zap.String("dst_client_id", pp.pathEnd2.info.ClientID),
-				zap.Error(err),
-			)
+			pp.logFailedTx(pp.pathEnd2.info, pp.pathEnd1.info, err)
 			return err
 		}
 		return nil
 	})
 	eg.Go(func() error {
 		if err := pp.assembleAndSendMessages(ctx, pp.pathEnd1, pp.pathEnd2, pathEnd2Messages); err != nil {
-			pp.log.Error("Error sending messages",
-				zap.String("src_chain_id", pp.pathEnd2.info.ChainID),
-				zap.String("dst_chain_id", pp.pathEnd1.info.ChainID),
-				zap.String("dst_client_id", pp.pathEnd1.info.ClientID),
-				zap.Error(err),
-			)
+			pp.logFailedTx(pp.pathEnd1.info, pp.pathEnd2.info, err)
 			return err
 		}
 		return nil
 	})
 	return eg.Wait()
 }
 
+func (pp *PathProcessor) logFailedTx(src, dst PathEnd, err error) {
+	if errors.Is(err, chantypes.ErrRedundantTx) {
+		pp.log.Debug("Packet(s) already handled by another relayer")
+		return
+	}
+	pp.log.Error("Error sending messages",
+		zap.String("src_chain_id", src.ChainID),
+		zap.String("dst_chain_id", dst.ChainID),
+		zap.String("src_client_id", src.ClientID),
+		zap.String("dst_client_id", dst.ClientID),
+		zap.Error(err),
```

Comment:
`channel-id` could be a nice addition here since there is the possibility that you are relaying on many channels between two chains and determining which channels are seeing failures can be helpful in debugging.

Reply:
With the PathEnd's channel filter list, we don't actually know which channel is having the issue from here since there may be messages for multiple channels. The logged error may include that field though.